hazelcast/src/main/java/com/hazelcast/spi/merge/HyperLogLogMergePolicy.java

```diff
@@ -17,9 +17,8 @@
 package com.hazelcast.spi.merge;
 
 import com.hazelcast.cardinality.impl.hyperloglog.HyperLogLog;
-import com.hazelcast.spi.SplitBrainMergeEntryView;
 
-import static com.hazelcast.spi.merge.SplitBrainMergePolicyDataSerializerHook.HYPER_LOG_LOG;
+import static com.hazelcast.spi.impl.merge.SplitBrainDataSerializerHook.HYPER_LOG_LOG;
```

Comment:
impl import in public class?

Reply:
It's an SPI internal import. I think we did this before.